engine/classes/Elgg/Database/Plugins.php

```diff
@@ -410,11 +402,10 @@ function find($status = 'active') {
 			'type' => 'object',
 			'subtype' => 'plugin',
 			'limit' => ELGG_ENTITIES_NO_VALUE,
-			'selects' => array('plugin_oe.*', 'ps.value'),
```

Comment:
You would have to update ElggPlugin and ElggWidget to not use private settings for former attributes. Another reason to loose private settings.